### PR TITLE
Update and refresh Create package dotnet

### PR DESCRIPTION
**Pull request by v-thepet**
User Story https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1973816.

### CODE REVIEW

**Comment by v-thepet** [comment]
@Mikejo5000 @nkolev92 @zivkan @JonDouglas - This article is ready for your review. The article has a couple of includes that I edited, but luckily it doesn't look like any other open PRs are trying to use these includes. Thanks for your timely review.

Note the warnings caused by invalid author/ms.author in two files that aren't changed in this PR. I started getting these warnings in all my open PRs as of last night 8/29. The author/ms.author are invalid in the upstream, but several other files in the repo also have these invalid names, and aren't being pulled in to PRs. I can't explain it. Please review the relevant article and includes, and I'll work on getting the extra files out of the PRs. Thanks.

File: docs/create-packages/creating-a-package-dotnet-cli.md [hunk 1]
```diff
@@ -1,48 +1,46 @@
...
+This topic applies only to .NET and other projects that use the [SDK-style format](../resources/check-project-format.md). For these projects, NuGet uses information from the project file to create a package. For quickstart tutorials, see [Create packages with the dotnet CLI](../quickstart/create-and-publish-a-package-using-the-dotnet-cli.md) or [Create packages with Visual Studio](../quickstart/create-and-publish-a-package-using-visual-studio.md).
 
-> [!IMPORTANT]
-> This topic applies to [SDK-style](../resources/check-project-format.md) projects, typically .NET Core and .NET Standard projects.
+The MSBuild `msbuild -t:pack` command is functionally equivalent to `dotnet pack`. To build with MSBuild, see [Create a NuGet package using MSBuild](creating-a-package-msbuild.md).
+
+> [!NOTE]
+> To create and publish packages for non-SDK-style projects, typically .NET Framework projects, see [Create a package using the nuget.exe CLI](Creating-a-Package.md) or [Create and publish a package using Visual Studio (.NET Framework)](../quickstart/create-and-publish-a-package-using-visual-studio-net-framework.md).
```
Comment (nkolev92) [inline docs/create-packages/creating-a-package-dotnet-cli.md:21]: I think we link the msbuild restore as well, as it's what we recommend using if you have migrated PackageReference projects.

File: docs/create-packages/creating-a-package-dotnet-cli.md [hunk 2]
```diff
@@ -1,48 +1,46 @@
...
 
 ## Set properties
 
-The following properties are required to create a package.
+To create a package, add the following required properties to your project file:
```
Comment (nkolev92) [inline docs/create-packages/creating-a-package-dotnet-cli.md:25]: Given that all these properties have defaults, I don't love the `required` wording, and I wonder if you have a recommendation on improving clarity. 

You can effectively do: 

`dotnet new classlib` and `dotnet pack` and you have a package.

File: docs/create-packages/creating-a-package-dotnet-cli.md [hunk 3]
```diff
@@ -56,80 +54,85 @@ The following example shows a simple, complete project file with these propertie
 </Project>
 ```
 
-You can also set the optional properties, such as `Title`, `PackageDescription`, and `PackageTags`, as described in [MSBuild pack targets](../reference/msbuild-targets.md#pack-target), [Controlling dependency assets](../consume-packages/package-references-in-project-files.md#controlling-dependency-assets), and [NuGet metadata properties](/dotnet/core/tools/csproj#nuget-metadata-properties).
+You can add optional properties, such as `Title`, `PackageDescription`, and `PackageTags`. You can also surface assets from dependencies directly in the package by using the `IncludeAssets` and `ExcludeAssets` attributes.
```
Comment (nkolev92) [inline docs/create-packages/creating-a-package-dotnet-cli.md:57]: I think a better description would be something along the lines of

When using the dotnet CLI, PackageReference in your project files are automatically converted dependencies for the created package. You can control what assets get included through <link to controlling` dependency assets.

**Review by nkolev92 — COMMENTED** [review]
This looks great!

File: docs/create-packages/creating-a-package-dotnet-cli.md (hunk 2 above)
Comment (v-thepet) [reply docs/create-packages/creating-a-package-dotnet-cli.md]: @nkolev92 - Updated - what do you think.

**Comment by v-thepet** [comment]
@nkolev92 - Made some changes, please review.

File: docs/create-packages/creating-a-package-dotnet-cli.md [hunk 4]
```diff
@@ -18,55 +18,51 @@ This topic applies only to .NET and other projects that use the [SDK-style forma
...
 </Project>
 ```
 
-You can add optional properties, such as `Title`, `PackageDescription`, and `PackageTags`. You can also surface assets from dependencies directly in the package by using the `IncludeAssets` and `ExcludeAssets` attributes.
+You can add other optional properties, such as `Title`, `PackageDescription`, and `PackageTags`.
+
+>[!NOTE]
+> For packages you build for public consumption, pay special attention to the `PackageTags` property. Tags help others find your package and understand what it does.
+
+The `dotnet pack` command automatically converts `PackageReference`s  in your project files to dependencies in the created package. You can control which assets to include through the `IncludeAssets` and `ExcludeAssets` tags. For more information, see [Controlling dependency assets](../consume-packages/package-references-in-project-files.md#controlling-dependency-assets).
```
Comment (nkolev92) [inline docs/create-packages/creating-a-package-dotnet-cli.md:57]:
```suggestion
The `dotnet pack` command automatically converts `PackageReference`s  in your project files to dependencies in the created package. You can control which assets to include through the `IncludeAssets`, `ExcludeAssets` and `PrivateAssets` tags. For more information, see [Controlling dependency assets](../consume-packages/package-references-in-project-files.md#controlling-dependency-assets).
```

Comment (nkolev92) [reply docs/create-packages/creating-a-package-dotnet-cli.md:57]: For context, PrivateAssets is something that controls whether a packagereference becomes a dependency in a resulting package, so calling it out makes it complete.

**Review by nkolev92 — APPROVED** [review]
This looks great!
1 mergeable suggestion. 

I can merge as soon as that change is made.

**Pull request merged**